templates/settings/store-form.php

```diff
@@ -248,31 +255,66 @@
-                        </label>
+                    <div class="dokan-form-group-container">
+                        <div class="dokan-form-group">
+                            <label class="day control-label" for="opening-time[<?php echo esc_attr( $day_key ); ?>][working_status]">
+                                <?php echo apply_filters( 'dokan_show_store_days', esc_html( dokan_get_translated_days( $day_key ) ), $status ); ?>
+                            </label>
+                            <label for="">
+                                <select name="<?php echo esc_attr( $day_key ); ?>[working_status]" class="dokan-on-off dokan-form-control">
+                                    <?php foreach ( $dokan_store_status as $status_key => $status_value ) : ?>
+                                        <option value="<?php echo esc_attr( $status_key ); ?>" <?php ! empty( $status ) ? selected( $status, $status_key ) : ''; ?> >
+                                            <?php esc_html_e( $status_value, 'dokan-lite' ); ?>
+                                        </option>
+                                    <?php endforeach; ?>
+                                </select>
+
+                                <?php
+                                /**
+                                 * @since 3.2.16
+                                 */
+                                do_action( 'after_dokan_status_selection', $status );
+                                ?>
+
+                            </label>
+                            <label for="opening-time" class="time" style="visibility: <?php echo isset( $status ) && $status == 'open' ? 'visible' : 'hidden'; ?>" >
+                                <?php
+                                    /* translators: %1$s is replaced with current day string, %2$s is replaced with placeholder string & %3$s is replaced with store opening time as a string value. */
+                                    $dokan_store_opening_entities = sprintf( '<input type="text" class="dokan-form-control opening-time" name="opening_time[%1$s]" id="opening-time[%1$s]" placeholder="%2$s" value="%3$s" >', esc_attr( $day_key ), esc_attr( '00:00' ), esc_attr( dokan_get_store_opening_time( $day_key ) ) );
+
+                                    echo apply_filters( 'dokan_store_opening_time_entities', $dokan_store_opening_entities, $day_key );
+                                ?>
+                            </label>
+                            <label for="closing-time" class="time" style="visibility: <?php echo isset( $status ) && $status == 'open' ? 'visible' : 'hidden'; ?>" >
```

Comment:
This for will not work in `<label for="opening-time" >`

Remember, why we use this `for`. We need the id in the input same name as this. Then, on click this label, the input will be focused. 

Talk with me, if you have any confusion.